pkg/cloud/aws/services/ec2/mock_ec2iface/ec2api.go

```diff
@@ -1,18 +1,4 @@
-/*
-Copyright 2018 The Kubernetes Authors.
 
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
```

Comment:
Is the removal here intentional?

Reply:
It was. But looking at other generated files, the headers are injected. So I will revert this change. In the process, I will also remove dangling `boilerplate` files.